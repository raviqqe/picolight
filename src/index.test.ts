import { expect, it } from "vitest";
import { highlight } from "./index.js";

it("highlights code", () => {
  expect(highlight("foo", []).textContent).toMatchSnapshot();
});
