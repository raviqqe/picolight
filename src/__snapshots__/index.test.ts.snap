// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`highlights code 1`] = `
<DocumentFragment>
  f
  o
  o
</DocumentFragment>
`;
